### DISCUSSION TITLE
Does all scripts are Win11 compatible?

### DISCUSSION THREAD

**Discussion by katastrophe92** [undergroundwires/privacy.sexy#217] [Q&A]
Hello and thanks for this nice debloat tool 😺 . I do not want to brake a fresh Win11 install that's why ask first. Does it have a script OS compatibility list? What I'm the most interested in is the Nvidia telemetry scripts . Thanks

**Comment by ItsIgnacioPortal** [comment]
You could setup virtualbox or vmware and test it yourself. There's bound to be more people who will have this same question
